docs/cloud-provider-specifics.md

```diff
@@ -139,7 +139,7 @@ To integrate Heptio Ark with GCP, you should follow the instructions below to cr
     gcloud config list
     ```
 
-    Store the `project` value from the results in the environment variable `$PROJECT_ID`.
+    Store the `project` value from the results in the environment variable `$CLOUDSDK_CORE_PROJECT`.
```

Comment:
@Bradamant3 fyi, it looks like GCP's project precedence is 1) `--project` flag, 2) `$CLOUDSDK_CORE_PROJECT` env var, 3) preconfigured default project (`gcloud config set core/project PROJECT`).

Do you think we should just use this env var? It has the effect of overriding the preconfigured default project, which may be undesirable or unexpected (based on our docs)(?). WDYT?